bin/uglifyjs

```diff
@@ -301,9 +301,19 @@ if (specified["in-situ"]) {
     process.stdin.setEncoding("utf8");
     process.stdin.once("data", function() {
         clearTimeout(timerId);
-    }).on("data", function(chunk) {
+    }).on("data", process.stdin.isTTY ? function(chunk) {
```

Comment:
was this to workaround issues in bun?

Reply:
Was playing around with `bun` &minus; but this oddity in Command Prompt is just good old Node.js TTY bugs on Windows.

Probably one of the many that has been archived and marked as `won't fix`, so might as well work around it while I'm here.